packages/mdx/mdx.1.8.1/opam

```diff
@@ -37,7 +37,7 @@ depends: [
   "alcotest" {with-test}
 ]
 build: [
-  ["dune" "subst"] {pinned}
+  ["dune" "subst"] {dev}
```

Comment:
This will need to be patched in dune's opam file generation as this is was the auto-generated build command!

Reply:
This is already patched since dune 2.7 but this requires at least `(lang dune 2.7)`

Reply:
This commit was also mostly to retrigger CI as most of the revdeps failed due to dependency cycles: https://github.com/ocaml/opam-repository/pull/18141
Usually I ignore those unless I need to modify opam file already.